src/reference/asciidoc/message-publishing.adoc

```diff
@@ -142,6 +142,8 @@ public class IntegrationConfiguration {
 ----
 ====
 
+Starting with version 5.1.3, the `<int:enable-publisher>` component, as well as `@EnablePublisher` annotation are supplied with the `proxy-target-class` and `order` attributes for tuning a `ProxyFactory` configuration.
```

Comment:
as well as **the** `@EnablePublisher` ...

`s/are supplied with the/have the/`

for tuning **the** `ProxyFactory`